pkgs/tools/X11/xssstate/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchgit
+, libX11
+, libXScrnSaver
+}:
+stdenv.mkDerivation rec {
+  name = "xssstate";
+  #
+  # Use the date of the last commit, since there were bug fixes after the 1.1
+  # release.
+  #
+  version = "20220924";
+  src = fetchgit {
+    url = "https://git.suckless.org/xssstate/";
```

Comment:
```suggestion
    url = "https://git.suckless.org/xssstate/";
    rev = "5d8e9b49ce2970f786f1e5aa12bbaae83900453f";
```

Reply:
Fixed as of e634c0bf6b66030b2238b01878a65a20f28a4e46.